### PR TITLE
Remove Knockout and AngularJS mentions from the docs

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: api-reference/10 UI Widgets/GridBase/1 Configuration/onToolbarPreparing.md [hunk 1]
```diff
@@ -26,12 +26,12 @@ The [options of the toolbar](/api-reference/10%20UI%20Widgets/dxToolbar/1%20Conf
...
-- <img src="/Content/images/doc/20_2/DataGrid/icons/toolbar_save.png" alt="DevExtreme HTML5 JavaScript jQuery Angular Knockout Widget DataGrid Toolbar SaveButton" style="margin:1px; vertical-align:middle"/> - *"saveButton"*   
-- <img src="/Content/images/doc/20_2/DataGrid/icons/toolbar_revert.png" alt="DevExtreme HTML5 JavaScript jQuery Angular Knockout Widget DataGrid Toolbar RevertButton" style="margin:1px; vertical-align:middle"/> - *"revertButton"*   
-- <img src="/Content/images/doc/20_2/DataGrid/icons/toolbar_export.png" alt="DevExtreme HTML5 JavaScript jQuery Angular Knockout Widget DataGrid Toolbar Exporting" style="margin:1px; vertical-align:middle"/> - *"exportButton"*     
-- <img src="/Content/images/doc/20_2/DataGrid/icons/toolbar_applyFilter.png" alt="DevExtreme HTML5 JavaScript jQuery Angular Knockout Widget DataGrid Toolbar ApplyFilterButton" style="margin:1px; vertical-align:middle"/> - *"applyFilterButton"*     
+- <img src="/Content/images/doc/20_2/DataGrid/icons/columnChooser.png" alt="DevExtreme HTML5 JavaScript jQuery Angular Widget DataGrid Toolbar ColumnChooserButton" style="margin:1px; vertical-align:middle"/> - *"columnChooserButton"*
```
Comment (artem-kurchenko) [inline api-reference/10 UI Widgets/GridBase/1 Configuration/onToolbarPreparing.md:29]: Why don't we mention React and Vue here in the alt attribute?

**Comment by artem-kurchenko** [comment]
We (Support Team) can approve this if we don't lose this old stuff on a new site(s). Can you confirm that this won't happen?)

**Comment by RomanTsukanov** [comment]
> 
> 
> We (Support Team) can approve this if we don't lose this old stuff on a new site(s). Can you confirm that this won't happen?)

There won't be a separate site for Knockout or AngularJS, so these examples won't be available in 20.2 and future versions. That's why I'm removing them from this version. Whether old versions (20.1 and older) will continue to exist is a moot point. My guess is they will in some form, but I may be wrong. Regardless of this, 20.1 and the examples will always be available on GitHub.

**Comment by RomanTsukanov** [comment]
We discussed these changes and agreed that they should wait until we decide where old documentation versions will go when we separate the site. The details are in [this Trello card](https://trello.com/c/3LTB99AK/1334-find-and-rewrite-all-the-mentions-of-the-template-in-ang-angjs-knockout).

For now, I'm closing this PR.

**Pull request closed without merging**